game-core/src/main/java/org/triplea/game/server/HeadlessGameServer.java

```diff
@@ -607,79 +586,41 @@ private static void usage() {
     // TODO replace this method with the generated usage of commons-cli
     log.info("\nUsage and Valid Arguments:\n"
         + "   " + TRIPLEA_GAME + "=<FILE_NAME>\n"
-        + "   " + TRIPLEA_SERVER + "=true\n"
         + "   " + TRIPLEA_PORT + "=<PORT>\n"
         + "   " + TRIPLEA_NAME + "=<PLAYER_NAME>\n"
         + "   " + LOBBY_HOST + "=<LOBBY_HOST>\n"
         + "   " + LOBBY_PORT + "=<LOBBY_PORT>\n"
-        + "   " + LOBBY_GAME_COMMENTS + "=<LOBBY_GAME_COMMENTS>\n"
-        + "   " + LOBBY_GAME_HOSTED_BY + "=<LOBBY_GAME_HOSTED_BY>\n"
-        + "   " + LOBBY_GAME_SUPPORT_EMAIL + "=<youremail@emailprovider.com>\n"
         + "   " + LOBBY_GAME_SUPPORT_PASSWORD + "=<password for remote actions, such as remote stop game>\n"
-        + "   " + LOBBY_GAME_RECONNECTION + "=<seconds between refreshing lobby connection [min "
-        + LOBBY_RECONNECTION_REFRESH_SECONDS_MINIMUM + "]>\n"
-        + "   " + MAP_FOLDER + "=mapFolder"
-        + "\n"
-        + "   You must start the Name and HostedBy with \"Bot\".\n"
-        + "   Game Comments must have this string in it: \"automated_host\".\n"
-        + "   You must include a support email for your host, so that you can be alerted by lobby admins when your "
-        + "host has an error."
-        + " (For example they may email you when your host is down and needs to be restarted.)\n"
-        + "   Support password is a remote access password that will allow lobby admins to remotely take the "
-        + "following actions: ban player, stop game, shutdown server."
-        + " (Please email this password to one of the lobby moderators, or private message an admin on the "
-        + "forums.triplea-game.org forum.)\n");
+        + "   " + MAP_FOLDER + "=<MAP_FOLDER>"
+        + "\n");
   }
 
   private static void handleHeadlessGameServerArgs() {
     boolean printUsage = false;
 
-    final File mapFolder = ClientSetting.mapFolderOverride.getValueOrThrow().toFile();
-    if (!mapFolder.isDirectory()) {
-      log.warning("Invalid '" + MAP_FOLDER + "' param, map folder must exist: '" + mapFolder + "'");
-      printUsage = true;
+    if (!ClientSetting.mapFolderOverride.isSet()) {
+      ClientSetting.mapFolderOverride.setValue(
+          Paths.get(SystemProperties.getUserHome(), "/triplea/downloadedMaps"));
```

Comment:
At a minimum, I'd recommend using `ClientFileSystemHelper#getUserRootFolder()` (see how `ClientSetting#userMapsFolderPath` is initialized).  You could probably just use `ClientSetting#userMapsFolderPath` itself.

Setting `ClientSetting#mapFolderOverride` also seems to duplicate the code in `ArgParser`.  Is it possible to have `ArgParser` set this default value if it doesn't encounter `MAP_FOLDER`?

Reply:
I rolled back the default value of maps folder (for now; it was rolled back to reduce complexity of this branch, leaving for a follow-up task)

Reply:
> Correction, went ahead with the defaulting code per the suggestion to use `ClientFileSystemHelper`.

I don't see a commit that implemented this.  Did you reconsider, or am I misunderstanding your intent?

Reply:
Thanks for the follow-up, I did not get this done completely. I pushed:  adb6d0568 just now to get the path from `ClientFileSystemHelper`.